### ISSUE TITLE
Improve links to code examples of component usages in "Usage Insights"

### ISSUE THREAD

**Issue by adamstankiewicz** [openedx/paragon#2239]
It appears the links to the code examples for component usages in consuming repos may not all be fully working (majority do!).

<img width="961" alt="image" src="https://user-images.githubusercontent.com/2828721/235146440-a77bfe27-0c9e-49c2-b246-52ebbb954529.png">

Examples:

* `edx-platform`
  * The links to `cms/**/*` and `xmodule/**/*` lead to a 404 page on GitHub, though the links to the `common/**/*`, `lms/**/*`, and `openedx/**/*` seem to work as expected.
* `frontend-app-support-tools`
  * All links lead to a 404 page on Github. Looks like the repository URL might be incorrect as `frontend-app-support`, not `frontend-app-support-tools` ([source](https://github.com/openedx/frontend-app-support-tools/blob/abec0340ac3a781284ba3e500db04b34aaa2b01e/package.json#L5-L8)).

```[tasklist]
### Acceptance Criteria
- [x] Investigate why some links seem to lead to a 404 page on GitHub, including the above examples and for any other projects, too.
- [x] If possible, resolve the issues to make the links work as expected. Note: may require a PR to consuming project repos.
- [x] Add a Segment event on click of a example code hyperlink (include clicked `url` and its associated project in event payload)
```

**Comment by PKulkoRaccoonGang** [comment]
PR: https://github.com/openedx/frontend-app-support-tools/pull/332

**Comment by viktorrusakov** [comment]
This was fixed in #2251 except for `edx-platform` links, created new issue #2282 to fix them separately (as it might be a much bigger lift)
